pkg/operator/status.go

```diff
@@ -289,9 +291,79 @@ func (optr *Operator) syncUpgradeableStatus() error {
 		coStatus.Message = "One or more machine config pools are updating, please see `oc get mcp` for further details"
 	}
 
+	// don't overwrite status if we didn't run
+	if supported, v := optr.isKubeletSkewSupported(); !supported {
+		coStatus.Reason = "KubeletSkewUnsupported"
```

Comment:
As is this would be overwriting a degraded and updating status, I think we probably only want to do this if we are available but not updating/degraded? Doesn't seem like this should preempt those more impt messages? wdyt